### DISCUSSION TITLE
All nodes disappearing on "hide selected nodes"

### DISCUSSION THREAD

**Discussion by KennyDenvers** [HEmile/juggl#107] [Q&A] [answered]
I'm very new to Juggl, just installed it tonight.
While I was playing around with it, it happened to me several times that all nodes disappeared after I clicked on "hide selected nodes". I thought I had selected only one individual node - but apparently I did not but instead all were selected. How does this happen?

Also: Is there a way simply to "undo" this? For me I closed the graph and reopened it. But is there a more "elegant" way?

**Comment by HEmile** [comment, marked as answer]
Yeah, this behavior is a bit unintuitive, and I might rework that someday. No promises though. The idea is that it removes nodes that are not next to any 'expanded' notes.

There is the Juggl Nodes pane on the right-bar pane of Obsidian. You can unhide nodes from there.

**Reply by KennyDenvers** [reply]
Ok, thanks! Will work from here. Thank you for your great work!
